### PR TITLE
use phpbench and change namespace

### PR DESCRIPTION
**Pull request by bendavies**
Hey,

I switched the custom benchmarking to use [PHPBench](https://phpbench.readthedocs.io/), and changed the namespace of the project at the same time.

Thoughts?

### CODE REVIEW

**Comment by goetas** [comment]
Wow! This to me looks great! 
Always wanted to see this benchmark running with phpbench!

@dunglas what do you think?

File: README.md [hunk 1]
```diff
@@ -41,37 +41,33 @@ $ docker-compose run --rm php composer install
...
 ``` bash
-$ docker-compose run --rm php bin/benchmark --horizontal-complexity 4
+$ docker-compose run --rm php ./vendor/bin/phpbench run --report=aggregate-sorted --iterations=10 --revs=10
 ```
 
-If you want to increase the vertical complexity of the serialization, you can use the `vertical-complexity` option 
-which represents a complexity factor:
+If you want to increase the horizontal complexity of the serialization, you can use the `parameters` option 
+which is an array of two integers, the first representing the horizontal and the second representing the vertical complexity.
+The defaults as horizontal=1, vertical=1.
```
Comment (goetas) [inline README.md]: What about putting the defaults to 10-10?

**Review by goetas — APPROVED** [review]

**Comment by dunglas** [comment]
Looks awesome! 👍

**Comment by goetas** [comment]
@bendavies please let me know when you are done with this, for me is ready to be merged! great work!

**Comment by bendavies** [comment]
yep, i'm done

**Comment by goetas** [comment]
great work!

**Pull request merged**